MAUI/DataGrid/Summaries.md

```diff
@@ -11,17 +11,647 @@ documentation: UG
+                                      Format="{}{Sum:c}"
+                                      MappingName="Salary"
+                                      SummaryType="Int32Aggregate" />
+            <sfGrid:DataGridSummaryColumn Name="ProductCount"
+                                      Format="{}{Count}"
+                                      MappingName="Salary"
+                                      SummaryType="CountAggregate" />
+            </sfGrid:DataGridSummaryRow.SummaryColumns>
+        </sfGrid:DataGridSummaryRow>
+</sfGrid:SfDataGrid.CaptionSummaryRow>
+{% endhighlight %}
+
+{% highlight c#%}
+DataGridSummaryRow summaryRow = new DataGridSummaryRow();
+summaryRow.Title = "Total Salary:{TotalSalary} for {ProductCount} members";
+summaryRow.ShowSummaryInRow = true;
+summaryRow.SummaryColumns.Add(new DataGridSummaryColumn()
+{
+    Name = "TotalSalary",
+    MappingName = "Salary",
+    Format = "{Sum:c}",
+    SummaryType = SummaryType.DoubleAggregate
+});
+summaryRow.SummaryColumns.Add(new DataGridSummaryColumn()
+{
+    Name = "ProductCount",
+    MappingName = "Salary",
+    Format = "{Count}",
+    SummaryType = SummaryType.CountAggregate
+});
+sfGrid.CaptionSummaryRow= summaryRow;
+{% endhighlight %}
+{% endtabs %}
+
+The following screenshot shows the outcome for both values of `ShowSummaryInRow` to `true`:
+
+![DataGrid with customized caption summary row](Images\Caption-Summary\ShowSummaryInRow1.png)
+
+
+### Displaying summary for column
+
+Display summary information in the column by setting the `DataGridSummaryRow.ShowSummaryInRow` to `false` and define summary columns. `SfDataGrid.DataGridSummaryColumn` is the object of [DataGridSummaryRow.SummaryColumns]() collection that contains the following important properties:
+
+* [Name](): Defines name of the `DataGridSummaryColumn` to denote the `DataGridSummaryColumn` in `DataGridSummaryRow` with title.
+* [MappingName](): Defines the corresponding column name used for the summary calculation.
+* [SummaryType](): Defines the `SummaryType` (enum) property to define the aggregate type for the summary calculation. 
+
+The `DataGrid` control provides the following predefined aggregates:
+
+  * CountAggregate
+  * Int32Aggregate
+  * DoubleAggregate
+
+[CustomAggregate]() defines the `CustomAggregate` class object when the summary type is set as `Custom` that calculates the custom summaries.
+
+The [Format]() defines the `string` property that formats the summary value and displays it. The `Format` property may contains two parts separated by a colon (:). First part denotes the aggregate function name, and second part denotes display format of the summary value.
+
+Refer to the [Formatting Summary](#_Formatting_Summary) section to know more about how to format summary, and [Aggregate Types](#_Aggregate_Types) section to know about different summary types.
+
+In the following code snippet, summary is defined for `Salary` column:
```

Comment:
To display summary information in a column, you need to set the DataGridSummaryRow.ShowSummaryInRow property to false and define summary columns using the SfDataGrid.DataGridSummaryColumn object. The DataGridSummaryColumn object has several important properties:

[Name](https://chat.openai.com/): This property defines the name of the DataGridSummaryColumn, which is used to identify the column in the DataGridSummaryRow and provide a title for it.
[MappingName](https://chat.openai.com/): This property defines the corresponding column name used for the summary calculation.
[SummaryType](https://chat.openai.com/): This property defines the SummaryType enum property, which determines the type of aggregate function used for the summary calculation.
The DataGrid control provides several predefined aggregates, such as CountAggregate, Int32Aggregate, and DoubleAggregate. When the summary type is set as Custom, you can define a CustomAggregate class object to calculate custom summaries.

The [Format](https://chat.openai.com/) property defines a string property that formats the summary value and displays it. The Format property can have two parts separated by a colon (:). The first part denotes the aggregate function name, and the second part denotes the display format of the summary value.

Please refer to the "Formatting Summary" section for more information on how to format the summary, and the "Aggregate Types" section to learn about the different summary types.

In the following code snippet, a summary is defined for the Salary column:

Reply:
Modified.